gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Table.java

```diff
@@ -322,14 +322,15 @@ public Builder toBuilder() {
   }
 
   @Override
-  public boolean equals(Object obj) {
-    return obj instanceof Table
+  public final boolean equals(Object obj) {
+    return this == obj
+        || obj instanceof Table
```

Comment:
Not sure I understand here. This PR is adding final to the `equals` method and removes it from classes.